vm/src/frame.rs

```diff
@@ -1207,26 +1211,30 @@ impl Frame {
 
 impl fmt::Debug for Frame {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        trace!("formatting stack");
```

Comment:
is this needed?

Reply:
Oh, whoops, that's left over from debugging the recursion. Thanks for catching that!